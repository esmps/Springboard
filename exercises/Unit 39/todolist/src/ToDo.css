.ToDo{
    list-style-type: none;
    display: flex;
    flex-direction: row;

}

.ToDo-button{
    height: 25px;
    width: 25px;
    line-height: 22px;
    background-color: rgb(237, 87, 87);
    border-radius: 15px;
    border: solid 1px black;
    margin: auto 2px;
    margin-right: 20px;
}