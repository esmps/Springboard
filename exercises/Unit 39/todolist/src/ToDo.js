import './ToDo.css'
function ToDo({id, text, handleRemove}){
    return (
       <li className="ToDo">
        <button className="ToDo-button" onClick={() => handleRemove(id)}>X</button>
        <p className="ToDo-text">{text}</p>
       </li>
    );
}
export default ToDo;