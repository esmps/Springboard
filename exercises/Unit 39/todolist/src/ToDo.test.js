import { render } from "@testing-library/react";
import ToDo from "./ToDo";

//smoke test
it("renders without crashing", function() {
  render(<ToDo />);
});

// snapshot test
it("matches snapshot", function() {
  const {asFragment} = render(<ToDo />);
  expect(asFragment()).toMatchSnapshot();
});