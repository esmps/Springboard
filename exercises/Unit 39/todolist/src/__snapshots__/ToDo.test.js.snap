// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li
    class="ToDo"
  >
    <button
      class="ToDo-button"
    >
      X
    </button>
    <p
      class="ToDo-text"
    />
  </li>
</DocumentFragment>
`;
